src/container/rpc.h

```diff
@@ -73,6 +73,9 @@
 	X(CONT_ATTR_SET,						\
 		0, &CQF_cont_attr_set,					\
 		ds_cont_op_handler, NULL),				\
+	X(CONT_ATTR_DEL,						\
+		0, &CQF_cont_attr_del,					\
+		ds_cont_op_handler, NULL),				\
```

Comment:
This is a protocol change so you need to bump the version number technically.  We don't need to worry about backwards compatibility yet luckily.

I suspect we've several of these so we'll wait until 1.2 and then verify every protocol version is different from 1.0 so this doesn't need to change now, but will be something to consider for future changes.

Reply:
Depends, you could add an RPC to the end and keep the same protocol version, and just check for DER_PROTO on the client when you call it but in this case it's in the middle so RPCs defined after this one will be using a different offset.